go.mod

```diff
@@ -159,7 +159,7 @@ require (
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
 	go4.org/intern v0.0.0-20220617035311-6925f38cc365 // indirect
-	go4.org/unsafe/assume-no-moving-gc v0.0.0-20220617031537-928513b29760 // indirect
+	go4.org/unsafe/assume-no-moving-gc v0.0.0-20230221090011-e4bae7ad2296 // indirect
```

Comment:
Is there a specific reason you updated this dependency?

Reply:
I was facing issue initially while running go generate.
Error pic attached.
As per the error i have added this dependency & able to run go generate

![image](https://user-images.githubusercontent.com/79891777/231092001-d4b85543-8218-49fe-852f-53a645153f7e.png)